components/mdx/CodeBlock.tsx

```diff
@@ -22,7 +22,14 @@ const CodeBlock: React.FC<Props> = ({ children, className }) => {
       {({ className, style, tokens, getLineProps, getTokenProps }) => (
         <pre
           className={className}
-          style={{ ...style, padding: '20px 20px 0 20px', marginBottom: 0 }}
+          style={{
```

Comment:
Is it a good idea to use inline styling here?

Reply:
In terms of consistency, no. But I think having to create a scss file only for this element is unnecessary. So, I just chose to add the css to the inline style.